example-16-compatibility-rules.yml

```diff
@@ -0,0 +1,181 @@
+groups:
+- name: node_exporter-16-bcache
```

Comment:
This should really be one rule group, so at least everything is misaligned in the same way.

Reply:
I don't think this will work performance wise on bigger nodes, some parallel execution will be necessary.

Reply:
Yes, we need explicit parallelism here.  This still keeps per-collector data in timestamp sync, which is good enough.